### PR TITLE
fix: rating type issues

### PR DESCRIPTION
**Pull request by magrinj**
This pull request addresses an issue related to the metadata type of the `RATING` field. It has been identified that the `RATING` field must comply with the GraphQL enum type naming convention, which stipulates adherence to the regular expression pattern `^[_A-Za-z][_0-9A-Za-z]+$`. To resolve this, we have reformatted the rating values to align with this pattern, now utilizing the format `RATING_`.

Additionally, the front-end has been updated to ensure compatibility with this change.

### CODE REVIEW

**Comment by charlesBochet** [comment]
While testing, I've faced the following error:
image

File: packages/twenty-server/src/metadata/field-metadata/dtos/options.input.ts [hunk 1]
```diff
@@ -13,11 +15,11 @@ export class FieldMetadataDefaultOptions {
   label: string;
 
   @IsNotEmpty()
-  @IsString()
+  @IsValidGraphQLEnumName()
```
Comment (charlesBochet) [inline packages/twenty-server/src/metadata/field-metadata/dtos/options.input.ts:18]: +1

File: packages/twenty-server/src/metadata/field-metadata/field-metadata.service.ts [hunk 2]
```diff
@@ -60,6 +66,21 @@ export class FieldMetadataService extends TypeOrmQueryService<FieldMetadataEntit
       throw new NotFoundException('Object does not exist');
     }
 
+    // Double check in case the service is directly called
+    if (isEnumFieldMetadataType(fieldMetadataInput.type)) {
```
Comment (charlesBochet) [inline packages/twenty-server/src/metadata/field-metadata/field-metadata.service.ts:70]: I feel there is something wrong here: isEnumFieldMetadataType does not make sense to me
I propose areEnumOptionsRequired() and Rating field should not be part of it.

It's weird to include Rating to then exclude it

Comment (magrinj) [reply packages/twenty-server/src/metadata/field-metadata/field-metadata.service.ts:70]: @charlesBochet I can create another util for that, it's just that `isEnumFieldMetadataType` already exist and is used in other places

**Comment by magrinj** [comment]
> While testing, I've faced the following error...

I'll take a look on that !

**Comment by magrinj** [comment]
@charlesBochet I've rebased the PR and doesn't have the issue actually

**Review by charlesBochet — APPROVED** [review]
Re-tested, and work like a charm. LGTM!

**Pull request merged**